src/physics.F90

```diff
@@ -1091,6 +1091,9 @@ subroutine create_fission_sites(p, i_nuclide, i_reaction, bank_array, size_bank)
     logical :: in_mesh                  ! source site in ufs mesh?
     type(Nuclide),  pointer :: nuc
 
+    ! If don't create fission sites, return directly
+    if (no_nu) return
```

Comment:
This check can be moved further up into the `sample_reaction` subroutine in physics.F90 so that `create_fission_sites` is just never called (in fact, I don't think `sample_fission` would need to be called either).